lib/smart_proxy_dns_infoblox/dns_infoblox_plugin.rb

```diff
@@ -2,7 +2,7 @@ module Proxy::Dns::Infoblox
   class Plugin < ::Proxy::Provider
     plugin :dns_infoblox, ::Proxy::Dns::Infoblox::VERSION
 
-    default_settings :username => 'infoblox', :password => 'infoblox', :dns_server => 'infoblox.my.domain'
+    default_settings :username => 'infoblox', :password => 'infoblox', :dns_server => 'infoblox.my.domain', :view => 'default'
```

Comment:
The documentation uses "dns_view", could you rename the parameter please?

Reply:
I'll do it.